src/mds/Locker.cc

```diff
@@ -129,9 +129,7 @@ void Locker::tick()
 
 void Locker::send_lock_message(SimpleLock *lock, int msg)
 {
-  for (map<mds_rank_t,unsigned>::iterator it = lock->get_parent()->replicas_begin();
-       it != lock->get_parent()->replicas_end(); 
-       ++it) {
+  for (compact_map<mds_rank_t,unsigned>::iterator it = lock->get_parent()->replicas_begin(); !it.end(); ++it) {
```

Comment:
Please keep these lines down to 80 characters. There are some others around too.